dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/Continuous/FirstOrderFilterLimitationDetection.mo

```diff
@@ -0,0 +1,68 @@
+within Dynawo.NonElectrical.Blocks.Continuous;
+
+/*
+* Copyright (c) 2022, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
+*/
+
+block FirstOrderFilterLimitationDetection "First order filter with absolute and rate limits, a freezing flag, and limitation detection flags"
+  import Modelica;
+  import Dynawo;
+
+  extends Dynawo.NonElectrical.Blocks.Continuous.FirstOrderFilter;
+
+  Modelica.Blocks.Interfaces.BooleanOutput fMax annotation(
+    Placement(visible = true, transformation(origin = {210, 60}, extent = {{-10, -10}, {10, 10}}, rotation = 0), iconTransformation(origin = {110, 60}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Interfaces.BooleanOutput fMin annotation(
+    Placement(visible = true, transformation(origin = {210, -60}, extent = {{-10, -10}, {10, 10}}, rotation = 0), iconTransformation(origin = {110, -60}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Logical.Greater greater annotation(
```

Comment:
Are you sure the greater are well connected ? Do we have fMax = true when we reach the upper limits ? Same questions for lessEqual

Reply:
I made a mistake, it is now corrected.